middleware.go

```diff
@@ -82,34 +88,37 @@ func Session(auth *jwtauth.JWTAuth, o *Options) func(next http.Handler) http.Han
 							eh(r, w, err)
 							return
 						}
+
 						if user != nil {
+							ctx = withUser(ctx, user)
+
+							sessionType = proto.SessionType_User
 							if isAdmin {
 								sessionType = proto.SessionType_Admin
-							} else {
-								sessionType = proto.SessionType_User
 							}
-							ctx = WithUser(ctx, user)
 						}
 					}
 
 					if adminClaim {
 						sessionType = proto.SessionType_Admin
-						break
 					}
 
 					if projectClaim > 0 {
 						projectID := uint64(projectClaim)
 						ctx = withProjectID(ctx, projectID)
 						sessionType = proto.SessionType_Project
 					}
+				case adminClaim:
```

Comment:
If you add this we accept a JWT with only admin claim, which is not correct.
I expect admin+address which is handled in the `accountClaim` block.